interbtc-spec/docs/source/spec/oracle.rst

```diff
@@ -186,38 +154,14 @@ Specification
 
 *Returns*
 
-* `u128` (aggregate) exchange rate value
-
-
-.. *Substrate*
-
-``fn getExchangeRate(origin) -> Result<u128, ERR_MISSING_EXCHANGE_RATE> {...}``
-
-*Errors*
-
-``ERR_MISSING_EXCHANGE_RATE``: the last exchange rate information exceeded the maximum delay acceptable by the oracle. 
-
-Preconditions
-.............
- 
-This function can be called by any participant to retrieve the BTC/DOT exchange rate as tracked by the BTC Parachain.
-
-Function Sequence
-.................
-
-1. Check if the current (UNIX) time minus the ``LastExchangeRateTime`` exceeds ``MaxDelay``. If this is the case, return ``ERR_MISSING_EXCHANGE_RATE`` error. 
-
-2. Otherwise, return the ``ExchangeRate`` from storage.
-
-
+* Fixed point exchange rate value.
```

Comment:
I think in other files we describe the return value as part as the postcondition, e.g. something  like "The fixed point exchange rate value MUST be returned by this function". 

Same for a couple of other functions below.

Reply:
Agree, should be:
```suggestion
*Preconditions*

* The ``LastExchangeRateTime`` MUST NOT be before the current time minus the ``MaxDelay``.

*Postconditions*

* MUST return the fixed point base exchange rate.
```